components/infobox/wikis/overwatch/infobox_person_player_custom.lua

```diff
@@ -80,10 +84,23 @@ function CustomInjector:parse(id, widgets)
 			}},
 		}
 	elseif id == 'history' then
-		table.insert(widgets, Cell{
-			name = 'Retired',
-			content = {_args.retired}
-		})
+		local nationalHistory = _args.nationalteams
+		local automatedHistory = TeamHistoryAuto._results({
+			convertrole = 'true',
+			player = _pagename
+		}) or ''
+		automatedHistory = tostring(automatedHistory)
+		if automatedHistory == EMPTY_AUTO_HISTORY then
+			automatedHistory = nil
+		end
+
+		if String.isNotEmpty(nationalHistory) or String.isNotEmpty(automatedHistory) then
+			return {
+				Title{name = 'History'},
+				Center{content = {nationalHistory}},
+				Center{content = {automatedHistory}},
+			}
+		end
```

Comment:
```suggestion
		table.insert(widgets, Cell{name = 'Retired', content = {_args.retired}})
		table.insert(widgets, Center{content = {_args.nationalteams}})
```

Reply:
I think it makes more sense for the order to be:

National History (If present)
Normal History
Retired

Otherwise this just looks awkward 
![image](https://user-images.githubusercontent.com/58013431/232555225-ef7ba23a-d2b2-43ac-afca-5b373178d239.png)

Reply:
```suggestion
		table.insert(widgets, Center{content = {_args.nationalteams}}, 1)
		table.insert(widgets, Cell{name = 'Retired', content = {_args.retired}})
```

then